README.rst

```diff
@@ -0,0 +1,10 @@
+###################
+ts_interlockmonitor
+###################
+
+Code for handling enclosure monitoring for the TunableLaser.
+This code is expected to run on raspberry pi that has a thermal scanner box(RS-232) and microphone(GPIO/USB) attached to it.
```

Comment:
"a raspberry pi" and there will not be a thermal scanner box

Reply:
Please process this comment from Russell.